cp22/hpi-fun2/furniture/problem.json

```
@@ -0,0 +1,7 @@
+{
+  "description": "Given parts of furniture and their dependencies to one another return the time needed to build them.",
```

We usually write something about the solution, like 
```suggestion
  "description": "longest path in DAG with vertex weights and story",
```